### PR TITLE
[Scope change] Search punch list item reference

### PR DESCRIPTION
**Pull request by Gustav-Eikaas**
# Description

Please include a summary of the change and which issue is fixed.
List any external dependencies that are required for this change.

Completes: AB#FILL_IN_YOUR_ISSUE_ID

## Checklist:

-   [x] I have performed a self-review of my own code.
-   [x] I have linked my DevOps task using the AB# tag.
-   [x] My code is easy to read, and comments are added where needed.
-   [ ] My code is covered by tests.

### CODE REVIEW

File: src/apps/ScopeChangeRequest/hooks/Search/useFAMSearch.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+interface FAMSearch {
+    searchFAM: (
+        searchValue: string,
+        type: FAMTypes,
+        signal?: AbortSignal
+    ) => Promise<TypedSelectOption[]>;
+}
+
+/**
+ * Hook for searching in ProCoSys
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/hooks/Search/useFAMSearch.ts:15]: rename

File: src/apps/ScopeChangeRequest/Components/DetailView/RelatedObjects/Punch/Punch.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import { useQuery } from 'react-query';
+
+import { ScopeChangePunch } from '../../../../types/scopeChangeRequest';
+import { isProduction } from '../../../../../../Core/Client';
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/DetailView/RelatedObjects/Punch/Punch.tsx:4]: alias

File: src/apps/ScopeChangeRequest/Components/DetailView/RelatedObjects/Systems/System.tsx [hunk 3]
```diff
@@ -29,21 +27,22 @@ export const System = ({ system }: SystemProps): JSX.Element => {
...
+            onClick={() =>
+                window.open(
+                    `https://${isProduction() ? 'procosys' : 'procosystest'
+                    }.equinor.com/JOHAN_CASTBERG/Completion#System|${system.procosysId}`,
+                    '_blank'
+                )
+            }
+            key={system.id}
+        >
             <Icon name="placeholder_icon" />
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/DetailView/RelatedObjects/Systems/System.tsx:40]: use folder icon

File: src/apps/ScopeChangeRequest/Components/SearchReferences/SelectedReference.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
+import { ClickableIcon } from '../../../../packages/Components/Icon';
```
Comment (Gustav-Eikaas) [inline src/apps/ScopeChangeRequest/Components/SearchReferences/SelectedReference.tsx:1]: alias or create alias

**Review by jaysencpp — APPROVED** [review]

**Pull request merged**